### PR TITLE
Add Comment for Debug Cmdlet

### PR DESCRIPTION
**Pull request by MainiSuruchi**

### CODE REVIEW

File: AzFilesHybrid/AzFilesHybrid.psm1 [hunk 1]
```diff
@@ -3660,7 +3660,8 @@ class CheckResult {
 function Debug-AzStorageAccountAuth {
     <#
     .SYNOPSIS
-    Executes a sequence of checks to identify common problems with Azure Files Authentication issues.  
+    Executes a sequence of checks to identify common problems with Azure Files Authentication issues.
+    This function is applicable for only ADDS authentication, Does not work for AADDS and AAD Kerberos.
```
Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3664]: We now call this Entra Kerberos

Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3664]: Does -> does

**Review by MaximeKjaer — CHANGES_REQUESTED** [review]

File: AzFilesHybrid/AzFilesHybrid.psm1 (hunk 1 above)
Comment (MainiSuruchi) [reply AzFilesHybrid/AzFilesHybrid.psm1]: fixed

Comment (MainiSuruchi) [reply AzFilesHybrid/AzFilesHybrid.psm1]: fixed

**Review by MaximeKjaer — APPROVED** [review]

**Pull request merged**